Add 5 Digit World Cup 2024 News Post (#10822)
ppy/osu-wiki: news/2024/2024-01-23-5-digit-world-cup-2024-registrations-now-open.md

## Changes From 2023

[5WC 2023](https://osu.ppy.sh/community/forums/topics/1699537?n=1) featured lots of improvements from previous iterations, but every rose has its thorns. This year we took a moment to reflect on how to further reduce points of discomfort/awkwardness that still exist in the series. For starters, last year's freemod rules proved to be too confusing for our liking.

Additionally, we found the notoriously-harsh BWS to be too restrictive. Player feedback noted that our qualifier procedure felt awkward by including a freemod map, so we revamped the qualifiers stage to be more comfortable and thorough.

### Key Changes

- 5WC 2024 now uses the standard [BWS](/wiki/Tournaments/Badge-weighted_seeding) formula and does not count tournament badges earned before Jan 1, 2021.
- The qualifiers procedure has changed: We now feature a pool with 14 maps at a harder difficulty target than before (6.5 stars).
- The qualifiers mappool will not feature a freemod map.
- This year's grand finals mappool is slightly more challenging at 7.0 stars.
- There is an all-new web-based registrations process!

## Registrations

> Players and organizers will need to login **[here](https://5wc.stagec.xyz/)** to register and/or manage their teams. Organizers have a new, private web interface they can use to manage their teams. Players who are selected will not know until told directly by their organizer.

Reviewer: ```suggestion
Players and organisers will need to log in **[here](https://5wc.stagec.xyz/)** to register or manage their teams. Organisers have a new, private web interface they can use to manage their teams. Players who are selected will not know until told directly by their organiser.
```

Additionally, prospective players must:

- Not be restricted or hold an active tournament ban by the end of registrations.
- Must not violate the 5WC 2024 community rules in a way which warrants disqualification.

> - Be within the rank range of #10,000-99,999 after the [BWS formula](/wiki/Tournaments/Badge-weighted_seeding) is applied with badges earned on or after 01-01-2021.

Reviewer: ```suggestion
- Be within the rank range of #10,000–99,999 after the [BWS formula](/wiki/Tournaments/Badge-weighted_seeding) is applied with badges earned on or after 1 January 2021.
```

- Join our [Discord server](https://discord.gg/WxX2UkdSMq).

### Tryouts

> Most countries require their players to tryout in order to be considered for the team. To check who your country's tryouts organizer is, check out [this spreadsheet](https://docs.google.com/spreadsheets/d/1AnmOB2qatI0wFF_yj0NLoAevHSRUI2AEA0Q-Csn268Y/edit#gid=0). You'll need to get in touch with the tryouts community for your country in order to participate. If you don't see your country on this list, join our [Discord server](https://discord.gg/WxX2UkdSMq) and reach out to the Admin team by opening a ticket in our #tickets channel, we'll be happy to help.

Reviewer: "tryout" is a noun ("try out" is the verb form)

```suggestion
Most countries require their players to pass tryouts in order to be considered for the team. To check who your country's tryout organiser is, check out [this spreadsheet](https://docs.google.com/spreadsheets/d/1AnmOB2qatI0wFF_yj0NLoAevHSRUI2AEA0Q-Csn268Y/edit#gid=0).

You'll need to get in touch with the tryouts community for your country in order to participate. If you don't see your country on this list, join our [Discord server](https://discord.gg/WxX2UkdSMq) and reach out to the admin team by opening a ticket in our `#tickets` channel, we'll be happy to help.
```

## Schedule

| Event | Date |
| --: | :-- |

> | Player Registrations | January 22 – February 11 (23:59 UTC) |
> | Team Submission Deadline | February 18 (23:59 UTC) |
> | Screening Buffer | Approximately 2 weeks |
> | Qualifiers Showcase | March 4 |
> | Qualifiers | March 8 - March 10 (23:59 UTC) |
> | Round of 32 | Match 16 - March 17 |
> | Round of 16 | March 23 - March 24 |
> | Quarterfinals | March 30 - March 31 |
> | Semifinals | April 6 - April 7 |
> | Finals | April 13 - April 14 |
> | Grand Finals | April 20 - April 21 |

Reviewer: ```suggestion
| Player registrations | January 22 – February 11 (23:59 UTC) |
| Team submission Deadline | February 18 (23:59 UTC) |
| Screening buffer | Approximately 2 weeks |
| Qualifier showcase | March 4 |
| Qualifiers | March 8 – March 10 (23:59 UTC) |
| Round of 32 | Match 16 – March 17 |
| Round of 16 | March 23 – March 24 |
| Quarterfinals | March 30 – March 31 |
| Semifinals | April 6 – April 7 |
| Finals | April 13 – April 14 |
| Grand Finals | April 20 – April 21 |
```

## Prizes

> At the time of writing, the community has donated over 500USD to our prize pool which will be distributed among the podium. Each player will receive an equal share of the prize pool allotted to the team.

Reviewer: ```suggestion
At the time of writing, the community has donated over $500 USD to our prize pool which will be distributed among the podium. Each player will receive an equal share of the prize pool allotted to the team.
```

![](/wiki/shared/news/2024-01-23-5-digit-world-cup-2024-registrations-now-open/5wc2024-prizes.png)

> We heavily encourage all eligible players with an interest in playing to tryout for your country. All are more than welcome to spectate and/or get involved with our community via our [Discord server](https://discord.gg/WxX2UkdSMq).

Reviewer: ```suggestion
We heavily encourage all eligible players with an interest in playing to check out the tryouts for your country. Everyone is more than welcome to spectate or get involved with our community via our [Discord server](https://discord.gg/WxX2UkdSMq).
```
